dkg-gadget/src/meta_async_rounds/dkg_gossip_engine/network.rs

```diff
@@ -154,19 +153,12 @@ const MAX_PENDING_MESSAGES: usize = 8192;
 #[allow(unused)]
 mod rep {
 	use sc_peerset::ReputationChange as Rep;
-	/// Reputation change when a peer sends us any message.
-	///
-	/// This forces node to verify it, thus the negative value here. Once message is verified,
-	/// reputation change should be refunded with `ANY_MESSAGE_REFUND`.
-	pub const ANY_MESSAGE: Rep = Rep::new(-(1 << 4), "Any message");
-	/// Reputation change when a peer sends us any message that is not invalid.
-	pub const ANY_MESSAGE_REFUND: Rep = Rep::new(1 << 4, "Any message (refund)");
 	/// Reputation change when a peer sends us a message that we didn't know about.
 	pub const GOOD_MESSAGE: Rep = Rep::new(1 << 7, "Good message");
-	/// Reputation change when a peer sends us a bad message.
-	pub const BAD_MESSAGE: Rep = Rep::new(-(1 << 12), "Bad message");
 	/// We received an unexpected message packet.
 	pub const UNEXPECTED_MESSAGE: Rep = Rep::new_fatal("Unexpected message packet");
+	/// Reputation change when a peer sends us the same message over and over.
+	pub const DUPLICATE_MESSAGE: Rep = Rep::new(-(1 << 12), "Duplicate message");
```

Comment:
The only issue here is that in certain parts of the protocol, we might intentionally send messages more than once.

Reply:
From the same peer? like the same peer sending us the same exact message (with the same hash) more than one time?

Also, in that case, if you can give me an example of when we will use that, we can add a simple atomic counter to keep track of how many times this peer sent us the exact same message, and if we reached `n` times we say ok this too much and maybe they are flooding the network so we start to add neg rep to them.